source/features/remove-is-open-is-closed.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+const countMatches = (string: string, regex: RegExp): number => {
+	return ((string || '').match(regex) ?? []).length;
+};
+
+function createMergeLink(): HTMLAnchorElement {
+	const linkMergedSearchParameters = new URLSearchParams(location.search);
+	const linkIsMerged: HTMLAnchorElement = (
+		<a href="" className="btn-link">
+		Merged
+		</a>
+	);
+	const regexpQueryTotal = /is:open|is:closed|is:issue/g;
+	const regexpQuery = /is:open|is:closed|is:issue/;
+
+	let linkMergedSearchString = new URLSearchParams(location.search).get('q') ?? select<HTMLInputElement>('#js-issues-search')!.value;
+
+	while (countMatches(linkMergedSearchString, regexpQueryTotal) > 1) {
+		linkMergedSearchString = linkMergedSearchString.replace(regexpQuery, '').trim();
+	}
+
+	if (countMatches(linkMergedSearchString, /is:merged/) === 1) {
+		linkMergedSearchParameters.set('q', linkMergedSearchString.replace(/is:merged/, 'is:issue').trim());
+		linkIsMerged.classList.add('selected');
+	} else if (countMatches(linkMergedSearchString, regexpQueryTotal) === 1) {
+		linkMergedSearchParameters.set('q', linkMergedSearchString.replace(regexpQuery, 'is:merged').trim());
+	}
+
+	linkIsMerged.search = String(linkMergedSearchParameters);
+
+	return linkIsMerged;
+}
+
+function init(): void {
+	const divTableListFiltersParent = select('div.table-list-filters');
+	const inputJsIssuesSearch = select<HTMLInputElement>('#js-issues-search');
+	if ((divTableListFiltersParent === null) || (inputJsIssuesSearch === null)) {
+		return;
+	}
```

Comment:
Only use this condition if there’s some situation where these elements are absent. If you always expect them there, don’t use this condition. 

This is connected to the `include` filter below. `isRepo` is too wide. Change the filter to this specific view, I think called `isDiscussionList`

Reply:
Wow, on `page-detect.ts` there is this whole list full of options to send to the include filter. Thanks.

Reply:
Would it be better if the page is isRepoIssueList? isDiscussionList has:
- isGlobalDiscussionList
  - https://github.com/issues
  - https://github.com/issues?q=is%3Apr+is%3Aopen
  - https://github.com/issues/assigned
  - https://github.com/issues/mentioned
  - https://github.com/pulls
  - https://github.com/pulls?q=issues
  - https://github.com/pulls/assigned
  - https://github.com/pulls/mentioned
  - https://github.com/pulls/review-requested
- isRepoDiscussionList
  - isRepoPRList
  - isRepoIssueList

isRepoIssueList is narrower

Reply:
All those pages have the same filters, so why exclude them? 😉

Reply:
What I meant here is that this feature should apply to all of them, hence `isDiscussionList` is appropriate.

07fb61b7

Reply:
What I meant here is that this feature should apply to all of them, therefore `isDiscussionList` is appropriate.

07fb61b7

Reply:
Thanks, you were clear on the comment. I did understood that isDiscussionList is appropriate.

Reply:
Ahhh, I saw again the code and the line you replaced by `isDiscussionList` I put it before reading your comment XD my mistake.